client/src/app/project-create-edit/components/project-metadata-form/project-metadata-form.component.ts

```diff
@@ -100,6 +99,10 @@ export class ProjectMetadataFormComponent implements OnInit, OnDestroy {
       overrideRequiredFields: {
         'project.content.contributors.project_role.text': false
       },
+      overrideGuidelines: {
+        'project.content.publications.official_hca_publication': 'Has the publication been accepted as an official HCA publication,\ ' +
```

Comment:
customised this to not affect Jacob's changes for Official HCA Publication field